tests/gnrc_ipv6_ext_frag/Makefile

```diff
@@ -0,0 +1,62 @@
+DEVELHELP := 1
+# name of your application
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-leonardo \
+                             arduino-mega2560 arduino-nano \
+                             arduino-uno hifive1 i-nucleo-lrwan1 mega-xplained \
+                             msb-430 msb-430h nucleo-f030r8 nucleo-f031k6 \
+                             nucleo-f042k6 nucleo-f070rb nucleo-f072rb \
+                             nucleo-f303k8 nucleo-f334r8 nucleo-l031k6 \
+                             nucleo-l053r8 saml10-xpro saml11-xpro \
+                             stm32f0discovery telosb thingy52 waspmote-pro \
+                             wsn430-v1_3b wsn430-v1_4 z1
```

Comment:
`stm32l0538-disco` and `hifive1b` should be added.

Reply:
First I check with the tests.